_redirects

```diff
@@ -39,16 +39,8 @@ layout: null
 /docs/v19.1/explore-the-admin-ui.html /docs/v19.1/admin-ui-overview.html 301
 /docs/v19.2/explore-the-admin-ui.html /docs/v19.2/admin-ui-overview.html 301
 /docs/v20.1/explore-the-admin-ui.html /docs/v20.1/admin-ui-overview.html 301
-/docs/v20.2/explore-the-admin-ui.html /docs/v20.2/admin-ui-overview.html 301
```

Comment:
admin-ui-overview.html does not exist under v20.2 and v21.1 folders. For those two versions the file is named ui-overview.html (added below on lines 42 and 43).